docs/configuration.asciidoc

```diff
@@ -214,14 +214,14 @@ Distributed tracing is enabled by default, however you can use this configuratio
 * *Type:* Array
 * *Default:* `[]`
 
-This option is an array of strings and determines which origins should be monitored as part of distributed tracing.
+This option is an array of strings or regexes and determines which origins should be monitored as part of distributed tracing.
 More specificly this option is consulted when the agent is about to add distributed tracing http header (`traceparent`) to a request.
-Please note that each item in the array should be a valid url and at least include the origin part (other parts of the url are ignored).
+Please note that this list testing the origin part of the url (other parts of the url are ignored), each item in the array should be a valid url or a regex to test the origin part.
```

Comment:
```suggestion
Please note that each item in the array should be a valid URL containing the origin (other parts of the url are ignored) or a RegExp object. If an item in the array is String, an exact match will be performed. If it's a RegExp object, its test function will be called with the request origin. 
```

Reply:
@bmorelli25 Can you have a look at the docs change? Thanks!

Reply:
I've slightly edited your above recommendation for clarity.

Reply:
<img width="777" alt="Screen Shot 2021-03-09 at 12 24 25 PM" src="https://user-images.githubusercontent.com/5618806/110533026-67545680-80d2-11eb-9320-5fd4fa33b726.png">

Huh, that's really neat. GitHub recognizes a suggestion edit as a contribution.